.github/workflows/go-tests.yml

```diff
@@ -96,29 +96,28 @@ jobs:
     - name: Notify Slack
       if: ${{ failure() }}
       run: .github/scripts/notify_slack.sh
-  # Temporarily changing until the situation with license headers in the generated code can be worked out
-  # check-generated-deep-copy:
-  #   needs:
-  #   - setup
-  #   runs-on: ${{ fromJSON(needs.setup.outputs.compute-large) }}
-  #   steps:
-  #   - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
-  #   # NOTE: This step is specifically needed for ENT. It allows us to access the required private HashiCorp repos.
-  #   - name: Setup Git
-  #     if: ${{ endsWith(github.repository, '-enterprise') }}
-  #     run: git config --global url."https://${{ secrets.ELEVATED_GITHUB_TOKEN }}:@github.com".insteadOf "https://github.com"
-  #   - uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4.0.1
-  #     with:
-  #       go-version-file: 'go.mod'
-  #   - run: make --always-make deep-copy
-  #   - run: |
-  #       if ! git diff --exit-code; then
-  #         echo "Generated code was not updated correctly"
-  #         exit 1
-  #       fi
-  #   - name: Notify Slack
-  #     if: ${{ failure() }}
-  #     run: .github/scripts/notify_slack.sh
+  check-codegen:
```

Comment:
Since this is now a general "make sure copywrite headers are correct everywhere" (by using the configuration in `.copwrite.hcl`, does it make sense to rename to `check-copywrite` since it will complain about files that are not related to code generation?

Reply:
It actually does both checking that copywrite headers are inserted and deep-copy generated files are up to date.
I was thinking of separating both but it make it hard, as we will never be able to check the end state which is deep-copy file are generated and all files have the headers (including the deep-copy generated ones)